cvmfs/options.h

```diff
@@ -15,24 +15,57 @@
 namespace CVMFS_NAMESPACE_GUARD {
 #endif
 
-namespace options {
+struct ConfigValue {
+  std::string value;
+  std::string source;
+};
+
+class OptionsManager {
+
+protected:
+  bool HasConfigRepository(const std::string &fqrn, std::string *config_path);
+
+public:
+  OptionsManager() :
+    config_(std::map<std::string, ConfigValue>()) {}
```

Comment:
It shouldn't be necessary to explicitly construct `config_`, its default constructor is anyway called.